h2o-r/h2o-package/R/frame.R

```diff
@@ -4671,12 +4814,16 @@ generate_col_ind <-function(data, by) {
 #' \dontrun{
 #' library(h2o)
 #' h2o.init()
-#' air <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv")
-#' groupCols <- c("Distance")
-#' sortCols <- c("IsArrDelayed", "IsDepDelayed")
-#' sortDirs <- c(TRUE, FALSE)
-#' h2o.rank_within_group_by(air, groupCols, sortCols, 
-#'                          sortDirs, new_col_name = "New_Rank", 
+#' 
+#' f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv"
```

Comment:
```suggestion
#' f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.zip"
```

Reply:
This file does not exist as csv -- its a zip file.  Fixing error found in build.